fastparquet/test/test_read.py

```diff
@@ -304,3 +304,15 @@ class MockIO:
         MockIO.loc = 0
         core.skip_definition_bytes(MockIO, num)
         assert len(block) == MockIO.loc
+
+
+def test_mr_times():
+    pf = fastparquet.ParquetFile(os.path.join(TEST_DATA, 'mr_times.parq'))
+    out = pf.to_pandas(mr_times=['date_added'])
+    expected = pd.to_datetime(
+            ["2016-08-01 23:08:01", "2016-08-02 23:08:02",
+             "2016-08-03 23:08:03", "2016-08-04 23:08:04",
+             "2016-08-05 23:08:04", "2016-08-06 23:08:05",
+             "2016-08-07 23:08:06", "2016-08-08 23:08:07",
+             "2016-08-09 23:08:08", "2016-08-10 23:08:09"])
```

Comment:
The test data that I have does not have that precision, and I don't have an impala or somesuch to conveniently write some more - I think its fine.